### ISSUE TITLE
Manual resync fails

### ISSUE THREAD

**Issue by jdalbey** [albinouty/shibby#36]
New shibby user here ... can't get it to run.   I'm trying to run shibby on Linux.  Have successfully installed dependency 'jq'. Running `./shibby` shows the help text.  Trying to do a manual resync produces a 'jq' parse error:
```
$ ./shibby -r
WARNING: no token found, requesting one and writing it to the token.id file
jq: parse error: Invalid numeric literal at line 1, column 6
```
As I look at the shibby code I see that resync is issuing a POST request to overdrive, so I tried doing that manually: 
```
$ payload=$(curl -X POST -f -s https://sentry-read.svc.overdrive.com/chip)
$ echo $payload

$ 

```
I don't know what the POST request is supposed to result in, but it's just getting a blank line, and I suppose 'jq' is failing because it expects something other than a blank line.
Not sure this is relevant but when I took a quick look at the Overdrive API (https://developer.overdrive.com/docs/api-security) it says the API use OAuth tokens, which is maybe different than what shibby is doing?

**Comment by albinouty** [comment]
Hey, the folks over at Overdrive/Libby have locked things down. Last I checked the authentication was working, but the fields present in the response payload required for shibby to work were no longer available. 

On top of this, Libby has changed how it enables devices other than phones to use your Libby credentials. Previously, it was anything that used the authentication code from the Libby app. However, now it appears that they only allow you to use another device if you have a passkey. And, shocker, the passkeys can only be created in a Libby app. This has effectively blocked tools like shibby from authenticating properly. 

In other words, I think shibby is dead. I haven't spent much time trying to resolve this so feel free to poke around.

**Comment by jdalbey** [comment]
Thanks for taking time to reply.  Sorry that shibby is dead.  I just created a new project that you might find of interest, [Libby Fetch](https://github.com/jdalbey/libbyfetch) takes a different approach.  I use browser automation to find out the name of the audiobook MP3 files and then download them.

**Comment by albinouty** [comment]
Very cool, thank you for the link. I am going to check it out. I'm in the process of writing a Chrome extension that will grab the mp3 files from the browser. Similar thought process to your Libby Fetch. I'm not sure when it will be done and the fact I have to use javascript isn't helping my motivation.
